Source/santad/DataLayer/WatchItems.mm

```diff
@@ -172,21 +157,21 @@ bool ConfirmValidWatchItemConfig(const NSDictionary *watch_item_dict) {
                                  PrefixTree<std::shared_ptr<WatchItemPolicy>> &tree,
                                  std::set<std::string> &paths) {
   for (const std::shared_ptr<WatchItemPolicy> &item : watch_items) {
-    glob_t g;
-    int err = glob(item->path.c_str(), 0, nullptr, &g);
+    glob_t *g = (glob_t *)alloca(sizeof(glob_t));
```

Comment:
This should be moved outside of the loop

Reply:
Done. I've also added a call to globfree to ensure any space allocated by glob() is freed at the end.

Reply:
Good catch on missing `globfree` - but I think it should be moved to inside the loop since we're not using the `GLOB_APPEND` flag

Reply:
Good call, done